src/main/java/net/apnic/rdap/resource/StaticResourceLocator.java

```diff
@@ -0,0 +1,62 @@
+package net.apnic.rdap.resource;
+
+import net.apnic.rdap.authority.RDAPAuthority;
+
+/**
+ * Class provides a static implementation of the ResourceLocator interface.
+ *
+ * This class will either always return the same RDAPAuthority or always
+ * throw a ResourceNotFoundException.
+ *
+ * @param <Resource> The type of resource this class assists with
+ */
+public class StaticResourceLocator<Resource>
```

Comment:
{Q} Is this just a wrapper around `ResourceLocator` to change the return/exception "contract" of the `ResourceLocator.authorityForResource` method?